src/cli/cli_network_data.cpp

```diff
@@ -751,6 +780,14 @@ otError NetworkData::OutputBinary(bool aLocal)
  * * Context ID
  * * Compress flag (`c` if marked or `-` otherwise).
  * @par
+ * Commissioning Dataset information is printed under `Commissioning` header:
+ * * Session ID if present in Dataset or `-` otherwise
+ * * Border Agent RLOC16 (in hex) if present in Dataset or `-` otherwise
+ * * Joiner UDP port number if present in Dataset or `-` otherwise
+ * * Steering Data (as hex bytes) if present in Dataset or `-` otherwise
+ * * Flags:
+ * * * e: If Dataset contains any extra unknown TLV
```

Comment:
There's an extra * here, for a sub list should be

`*   * e: If Dataset contains any extra unknown TLV`

Reply:
Thanks. Fixed in new push.